src/js/tabs/osd.js

```diff
@@ -909,7 +909,16 @@ OSD.constants = {
     CRASH_FLIP_MODE: {
       name: 'CRASH_FLIP_MODE',
       desc: 'osdWarningCrashFlipMode'
+    },
+    ESC_FAIL: {
+      name: 'OSD_WARNING_ESC_FAIL',
+      desc: 'osdWarningEscFail'
+    },
+    ESC_FAIL2: {
```

Comment:
What do we need this one for?

Reply:
Ahhhh!! This was a test that I forgot to remove!! I will update it asap 😱